.github/workflows/continuous-integration.yml

```diff
@@ -24,10 +24,10 @@ jobs:
       - name: Checkout git repository 🕝
         uses: actions/checkout@v2
 
-      - name: Set up Python 3.7 🐍
-        uses: actions/setup-python@v1
+      - name: Set up Python 3.10 🐍
+        uses: actions/setup-python@b55428b1882923874294fa556849718a1d7f2ca5
         with:
-          python-version: 3.7
+          python-version: 3.10.8
```

Comment:
why pin this specific version instead of relying on the latest micro available?

Reply:
The CI seems to read `3.10` as `3.1`, like [here](https://github.com/RasaHQ/rasa-sdk/actions/runs/3496484220/jobs/5854446771#step:3:3), and I didn't know another alternative to fix this.

Reply:
A quick search online seems to suggest to wrap this in single quotes as '3.10' instead, I'll try that instead 🤞🏻

Reply:
yep